beef/bump.go

```diff
@@ -98,6 +110,28 @@ func findLeafByOffset(offset uint64, bumpLeaves []BUMPLeaf) *BUMPLeaf {
 	return nil
 }
 
+func calculateFromChildren(offset uint64, bumpLeaves []BUMPLeaf) (*BUMPLeaf, error) {
+	offsetChild := offset * 2
+	offsetChildPair := offsetChild + 1
+	leaf := findLeafByOffset(offsetChild, bumpLeaves)
+	if leaf == nil {
+		return nil, errors.New("could not find child")
+	}
+	leafInPair := findLeafByOffset(offsetChildPair, bumpLeaves)
+	if leafInPair == nil {
+		return nil, errors.New("could not find child")
+	}
+	leftNode, rightNode := prepareNodes(*leaf, offset, *leafInPair, offsetChildPair)
+	str, err := merkleTreeParentStr(leftNode, rightNode)
+	if err != nil {
+		return nil, errors.New("could not find pair")
+	}
+	return &BUMPLeaf{
+		Hash:   str,
+		Offset: offset,
+	}, nil
+}
+
```

Comment:
Do you think it would be a good idea to extract these errors into external variables and add some test cases to cover these edge cases? 🤔😄 Additionally, small code style suggestion to use new lines after statements like `if' etc.

Reply:
New lines after 'if's introduced. 
In near future we will be swapping functions regarding BUMP calculation to go-SDK ones.
Because this is a temporary fix, and such situations can occur in theory and I'm not sure if they can happen in a real life scenario, and it can be time-consuming to find the hexes for which we get such errors. 
Therefore, I would hold off on developing any test cases to get them.